tests/schema/models.py

```diff
@@ -109,6 +109,13 @@ class Meta:
         apps = new_apps
 
 
+class BookWithChoices(models.Model):
+    select = models.CharField(choices=(("1", "One"), ("2", "Two")), max_length=10)
+
+    class Meta:
+        apps = new_apps
+
+
```

Comment:
We can reuse one of existing models e.g. `Author`:
```python
    def test_alter_field_choices_noop(self):
        with connection.schema_editor() as editor:
            editor.create_model(Author) 
        old_field = Author._meta.get_field("name")
        new_field = CharField(
            max_length=255,
            choices=[("Joe", "Joe"), ("Tom", "Tom")],
        )   
        new_field.set_attributes_from_name("name")
        with connection.schema_editor() as editor, self.assertNumQueries(0):
            editor.alter_field(Author, old_field, new_field, strict=True)
        with connection.schema_editor() as editor, self.assertNumQueries(0):
            editor.alter_field(Author, new_field, old_field, strict=True)
```
We try to avoid adding separate models for each test.

Reply:
That's so much cleaner thank you :+1: